types/jsonwebtoken/index.d.ts

```diff
@@ -83,6 +83,14 @@ export interface SignCallback {
     (err: Error, encoded: string): void;
 }
 
+export interface SecretCallback {
```

Comment:
Don't think this deserves its own type declaration. Can just do:
```ts
export type SecretFunc = (header: object, callback: (err: Error, secretOrPrivateKey: string | buffer) => void) => void;
```

Reply:
That's a good one 👍. Will update it soon!

Reply:
Forgot to remove this? It's already been inlined into `SecretFunc` so should be OK to delete.